torch/_functorch/_aot_autograd/dispatch_and_compile_graph.py

```diff
@@ -51,7 +51,7 @@ def _create_graph(f, args, *, aot_config: AOTConfig) -> torch.fx.GraphModule:
         export=aot_config.is_export,
         # Allow token discovery for joint fn tracing as tokens can be used in backward.
         _allow_token_discovery=True,
-    ):
+    ), dynamo_timed("_create_graph", log_pt2_compile_event=True):
```

Comment:
do we want a more detailed name for this or is `_create_graph` something generic?

Reply:
I figured there was some convention for naming these after the function they're called in (@jamesjwu are there any conventions we should be following for the naming?). Otherwise, totally agreed - I can name it something clearer like "aotdispatcher_makefx")

Reply:
Can we call this something like `dispatch_and_compile_graph._create_graph` to be more specific?